torchvision/transforms/v2/functional/_misc.py

```diff
@@ -99,6 +106,41 @@ def _get_gaussian_kernel2d(
     return kernel2d
 
 
+def _validate_kernel_size_and_sigma(
```

Comment:
Should we use `_validate_kernel_size_and_sigma` in `gaussian_blur_image` since we extracted the logic from here?